src/coins.ts

```diff
@@ -320,6 +324,7 @@ export const coins = CoinMap.fromCoins([
   erc20('xcd', 'CapdaxToken', 18, '0xca00bc15f67ebea4b20dfaaa847cace113cc5501', UnderlyingAsset.XCD),
   erc20('xin', 'Mixin', 18, '0xa974c709cfb4566686553a20790685a47aceaa33', UnderlyingAsset.XIN),
   erc20('xrl', 'Rialto', 9, '0xb24754be79281553dc1adc160ddf5cd9b74361a4', UnderlyingAsset.XRL),
+  erc20('yng', 'Young Token', 18, '0xa26cbb76156090f4b40a1799a220fc4c946afb3', UnderlyingAsset.YNG),
```

Comment:
Where is this underlying asset?

Reply:
https://bitgoinc.atlassian.net/browse/ERC20-151